libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -60,26 +64,29 @@ export class DhAdminUserManagementDataAccessApiStore extends ComponentStore<DhUs
 
   readonly getUsers = this.effect((trigger$: Observable<void>) =>
     trigger$.pipe(
+      withLatestFrom(this.state$),
       tap(() => {
         this.resetState();
 
         this.setLoading(LoadingState.LOADING);
       }),
-      switchMap(() =>
-        this.httpClient.v1MarketParticipantUserOverviewGet().pipe(
-          tapResponse(
-            (users) => {
-              this.setLoading(LoadingState.LOADED);
-
-              this.updateUsers(users);
-            },
-            () => {
-              this.setLoading(LoadingState.LOADED);
-
-              this.handleError();
-            }
+      switchMap(([, state]) =>
```

Comment:
Q: ever seen [ before for unused variable. Normally it is _ any history to this conversion?

Reply:
:) Well. the `[` is the opening square bracket of the array. It's the `,` part that is the magic. It allows us to skip the first element and not assign it to anything since it's not going to be used. Then the second element is assigned to `state`.

Reply:
So it's array destructuring but ignoring elements at the same time.

![image](https://user-images.githubusercontent.com/1096332/208069005-891c7987-5af1-4c49-aeeb-3b58cab2a47f.png)